contrib/localnet/scripts/start-zetaclientd.sh

```diff
@@ -43,7 +43,7 @@ then
      set_sepolia_endpoint
     fi
 
-    zetaclientd start < /root/password.file
+    zetaclientd-supervisor start < /root/password.file
```

Comment:
In theory with this change, we should be able to remove `restart-zetaclient` script used in local upgrade test, right?

Reply:
yes exactly